composer.json

```diff
@@ -18,8 +18,7 @@
         "symfony/polyfill-ctype": "^1.8"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.6",
-        "sebastian/version": "^1.0.1"
+        "phpunit/phpunit": "^4.8.36|| ^7.5.13"
```

Comment:
I don't really see a purpose in allowing phpunit versions 5 or 6 to be installed.

Version 8 is a bit harder to do, as the `teardownAfterClass` has a void return type, so we would have to have that as well.

Therefore it  makes sense to install phpunit 7.5, and work with that